school-library/rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_reader :person, :book
+  attr_accessor :date
+
+  def initialize(date, person, book)
+    @date = date
+    @person = person
+    person.rental << self
+    @book = book
+    book.rental << self
+  end
```

Comment:
- [ ] The association here it's correct! :clap: I just think we might want to have some methods on the `person` class and the `book` class that we can use to add new rentals.